src/quo2/components/tags/base_tag.cljs

```diff
@@ -2,31 +2,27 @@
   (:require
    [quo.react-native :as rn]))
 
-(defn style-container [size disabled border-color border-width background-color label type]
+(defn style-container [size disabled? border-color border-width background-color labelled? type]
   (merge {:height             size
           :border-color       border-color
           :background-color   background-color
           :border-width       border-width
           :border-radius      size
           :align-items        :center
           :justify-content    :center}
-         (when disabled
+         (when disabled?
            {:opacity 0.3})
-         (when (and (or (= type :icon) (= type :emoji)) (not label))
+         (when (and (or (= type :icon) (= type :emoji)) (not labelled?))
            {:width       size})))
 
 (defn base-tag
-  "opts
-   {:type :icon/:emoji/:label/:permission
-    :size 32/24}
-    :labelled true"
   [_]
-  (fn [{:keys [id size disabled border-color border-width background-color on-press
-               accessibility-label label type] :or   {size 32}} children]
-    [rn/touchable-without-feedback (merge {:disabled            disabled
+  (fn [{:keys [id size disabled? border-color border-width background-color on-press
+               accessibility-label labelled? type] :or   {size 32}} children]
+    [rn/touchable-without-feedback (merge {:disabled?            disabled?
```

Comment:
hmm, are you sure this is safe?
the :disabled key here is being passed to `rn/touchable-without-feedback` and the prop name there is `disabled` .. so does this work?
https://reactnative.dev/docs/touchablewithoutfeedback#disabled

Reply:
weird coz it actually disables but i am going to change this back to `disabled` instead of `disabled?` just to be on the safe side